lib/collection/src/shards/queue_proxy_shard.rs

```diff
@@ -60,9 +62,15 @@ impl QueueProxyShard {
         wrapped_shard: LocalShard,
         remote_shard: RemoteShard,
         wal_keep_from: Arc<AtomicU64>,
+        progress: Option<Arc<ParkingMutex<TransferTaskProgress>>>,
```

Comment:
Do you want to pass 'None' for testing somewhere?

Reply:
We don't.

I didn't pass it in shard snapshot transfer, but I do no, so we can remove the option.

To be honest, I do like not being dependent on a progress tracker, but if we ever want that again we can simply make it optional again.

Reply:
Done.